tests/test_helpers.sh

```
@@ -0,0 +1,170 @@
+}
+
+print_error() {
+    echo -e "${RED}$1${NC}"
+}
+
+print_warning() {
+    echo -e "${YELLOW}$1${NC}"
+}
+
+print_info() {
+    if [[ "$VERBOSE" == "true" ]]; then
```

Shellcheck (0.10.0)

[warning] VERBOSE is referenced but not assigned.

(SC2154)